### ISSUE TITLE
Sort clickable icons are hidden (behind of column's headline-text) if table has class table-sm

### ISSUE THREAD

**Issue by dpastov** [wenzhixin/bootstrap-table#4742]
**Short and descriptive example bug report title**

When you apply class 'table-sm' on your table while you have sort icons enabled and column headers are aligned to right - then sort icons are behind of text (due to overwritten padding-right: 30px).

**A summary of the issue and the browser/OS environment in which it occurs**

It will happen in all browsers since it's related to CSS>

**STR**
1. Apply table-sm on your table.
2. Make columns sortable
3. Align title to right or just make column width small (so headline takes all space).

Actual:
sort icons are behind of text.
Expected: 
padding-right: 30px is applied.

It happens due to 

```
.bootstrap-table .fixed-table-container .table.table-sm .th-inner {
    padding: .3rem;
}
```
and it overwrites padding-right: 30px

```
.bootstrap-table .fixed-table-container .table thead th .sortable {
    cursor: pointer;
    background-position: right;
    background-repeat: no-repeat;
    padding-right: 30px
}
```

**Example**

```
<table data-toggle="table"class="table-striped table-sm">
  <thead>
    <tr>
      <th data-sortable="true" >Headline 1</th>
      <th data-sortable="true" >Headline very long just to eat as much as possible space. Make it longer so it eats all the space of screen.</th>
      <th data-align="right" data-sortable="true">Headline 2</th>
      <th data-align="right" data-sortable="true">Headline 3</th>
    </tr>
  </thead>
</table>
```

Headline 2 and Headline 3 will have sort icons under text.
Headline 1 (if your screen is small) will have sort icons under text.

Live example
https://live.bootstrap-table.com/code/dpastov/1367

**Comment by wenzhixin** [comment]
Please provide an [Online Example](https://live.bootstrap-table.com) to show your problem, thanks!

**Comment by dpastov** [comment]
Live Example

https://live.bootstrap-table.com/code/dpastov/1367

**Comment by wenzhixin** [comment]
Thanks for the feedback, I think it's a bug.

**Comment by dpastov** [comment]
I fixed it by adding !important

```
.bootstrap-table .fixed-table-container .table thead th .sortable{
cursor:pointer;
background-position:right;
background-repeat:no-repeat;
padding-right:30px !important
}
```

**Comment by wenzhixin** [comment]
Fixed in #4750
